src/app/containers/LendBorrowSovryn/BorrowingContainer/index.tsx

```diff
@@ -58,6 +58,7 @@ const BorrowingContainer: React.FC<Props> = ({ currency }) => {
     const loanPool = LendingPoolDictionary.get(currency);
     const options = loanPool
       .getBorrowCollateral()
+      .filter(asset => asset !== Asset.SOV)
```

Comment:
@soulBit borrow asset list is defined here:

https://github.com/DistributedCollective/Sovryn-frontend/commit/abf5f6d79c73fa250c9fb1d771d6e09322731e9f#diff-1ce769a0a715c94345982d804949c15435dd7cd967d76bde09293a885d04d653R8

You can revert this filter back.

I made a fix for it some time ago and somehow didn't push it :/